src/ConfigManagers/Omissions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace LangleyFoxall\LaravelAuthenticationLog\ConfigManagers;
+
+class Omissions
+{
+    public static function omitCredentials($credentials)
+    {
+        $credentialsToOmit = config('auth-log.credentialsToOmit');
+
+        if ($credentialsToOmit == []) {
+            return $credentials;
+        }
+
+        else {
+            return array_diff_key($credentials, array_flip($credentialsToOmit));
```

Comment:
I would have done this differently, using `array_filter` and the `ARRAY_FILTER_USE_KEYS` constant